rfcs/0000-rename-unstable.md

```diff
@@ -0,0 +1,127 @@
+---
+feature: rename-unstable
+start-date: 2023-06-16
+author: @K900
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Users often interpret Nixpkgs "unstable" as "contains bugs", not the intended meaning of "can have breaking changes".
+To avoid this confusion, we should rename the "unstable" branch to something that doesn't have those connotations.
+
+# Motivation
+[motivation]: #motivation
+
+When introducing people to Nix and NixOS, the very first question that often arises is "why are we using unstable?".
+This is because the word "unstable" has two somewhat overloaded meanings - end users usually think about _application stability_
+(i.e., roughly, the "WTFs per hour"), whereas developers think about _API stability_. This results in a lot of confusion,
+especially for new users who are often told to "don't worry, just run unstable", which is a pretty scary prospect given the
+initial assumption of what "unstable" really refers to.
+
+# Detailed design
+[design]: #detailed-design
+
+## Aside: on the naming of cats
+
+This RFC intentionally does _not_ propose a new name to replace "unstable". The intent of this RFC is to not focus on the exact naming,
+but establish consensus over the _idea_ of renaming, and, if consensus is achieved, identify the technical hurdles.
+
+As such, following [in the footsteps of our ancestors](https://doc.rust-lang.org/std/ops/struct.Yeet.html), the placeholder new name
+for the branch shall henceforth be `yeet`.
+
+## Action plan
+
+1. Set up new branches in the Nixpkgs repository and new channel endpoints at `channels.nixos.org`, containing the same code as `nixpkgs-unstable`
+and `nixos-unstable`, but named `nixpkgs-yeet` and `nixos-yeet`.
```

Comment:
yeet is IMO worse than unstable and the meaning I found for it on the web is also not a good fit.

```
Yeet is a slang word that functions broadly with the meaning “to throw,” but is especially used to emphasize forcefulness and a lack of concern for the thing being thrown.
```

source https://www.merriam-webster.com/words-at-play/yeet-meaning-and-history


so let me create a sentence with it: NixOS got yeeted in my face and I lost all my teeth.

Reply:
`yeet` is a placeholder, not the final proposed name.